### PR TITLE
Updated getting stated card with helpful next steps

### PR DESCRIPTION
**Pull request by sejal27**
Now that we have a couple of more components like Heading and the ability to link to docs, here's an updated CRM card with helpful instructions for the next steps after deploying the first card.

image

**TODO**
- [ ] Review by Erin+Mark
- [ ] Update readme with more details and links to docs
- [ ] Update docs based on changes in file names and screenshots of the get started projects

### CODE REVIEW

File: src/app/app.functions/example-card.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+  const { firstname } = context.propertiesToSend;
+
+  const introMessage = {
+    type: "alert",
+    title: "Your first UI extension is ready!",
+    variant: "success",
+    body: {
+      type: "text",
+      format: "markdown",
+      text: "Congratulations! You just deployed your first **HubSpot UI extension**. This example CRM card uses the [ZenQuotes public API](https://zenquotes.io/) to display a daily quote. It also includes custom button actions using serverless functions.",
```
Comment (markhazlewood42) [inline src/app/app.functions/example-card.js:15]: >Congratulations! You just deployed your first **HubSpot UI extension**. This example CRM card uses the [ZenQuotes public API](https://zenquotes.io/) to display a daily quote. It also includes custom button actions built using serverless functions.

File: src/app/app.functions/example-card.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      distance: "small",
+    },
+    {
+      type: "heading",
+      text: "Next Steps",
+    },
+    {
+      type: "text",
+      format: "markdown",
+      text: "You can explore [more components](https://app.hubspot.com/l/extensible-ui/), or try code samples. When you get stuck, look at your [build or deploy logs](https://app.hubspot.com/l/developer-projects/) or [serverless functions logs](https://app.hubspot.com/l/private-apps/).",
```
Comment (markhazlewood42) [inline src/app/app.functions/example-card.js:31]: >You can explore more UI components or code examples in the [CRM card builder](https://app.hubspot.com/l/extensible-ui/). If you get stuck, take a look at your [build and deploy logs](https://app.hubspot.com/l/developer-projects/) or [serverless functions logs](https://app.hubspot.com/l/private-apps/).

**Review by markhazlewood42 — COMMENTED** [review]
A few suggested changes to copy.

**Review by brandenrodgers — APPROVED** [review]
lgtm! 👌

**Pull request merged**